subsys/bluetooth/audio/aics.c

```diff
@@ -67,34 +67,39 @@ static ssize_t read_description(struct bt_conn *conn,
 	BT_GATT_SECONDARY_SERVICE(BT_UUID_AICS),                               \
 	BT_GATT_CHARACTERISTIC(BT_UUID_AICS_STATE,                             \
 			       BT_GATT_CHRC_READ | BT_GATT_CHRC_NOTIFY,        \
-			       BT_GATT_PERM_READ_ENCRYPT,                      \
+			       BT_GATT_PERM_READ_ENCRYPT | BT_GATT_PERM_READ_128B_ENTROPY,   \
 			       read_aics_state, NULL, &_aics),                 \
 	BT_GATT_CCC(aics_state_cfg_changed,                                    \
-		    BT_GATT_PERM_READ | BT_GATT_PERM_WRITE_ENCRYPT),           \
+		    BT_GATT_PERM_READ | BT_GATT_PERM_WRITE_ENCRYPT                 \
+			 | BT_GATT_PERM_READ_128B_ENTROPY | BT_GATT_PERM_WRITE_128B_ENTROPY) \
 	BT_GATT_CHARACTERISTIC(BT_UUID_AICS_GAIN_SETTINGS,                     \
 			       BT_GATT_CHRC_READ,                              \
-			       BT_GATT_PERM_READ_ENCRYPT,                      \
+			       BT_GATT_PERM_READ_ENCRYPT | BT_GATT_PERM_READ_128B_ENTROPY    \
+				    | BT_GATT_PERM_WRITE_128B_ENTROPY,                   \
 			       read_aics_gain_settings, NULL, &_aics),         \
 	BT_GATT_CHARACTERISTIC(BT_UUID_AICS_INPUT_TYPE,                        \
 			       BT_GATT_CHRC_READ,                              \
-			       BT_GATT_PERM_READ_ENCRYPT,                      \
+			       BT_GATT_PERM_READ_ENCRYPT | BT_GATT_PERM_READ_128B_ENTROPY    \
+				    | BT_GATT_PERM_WRITE_128B_ENTROPY,                   \
 			       read_type, NULL, &_aics),                       \
 	BT_GATT_CHARACTERISTIC(BT_UUID_AICS_INPUT_STATUS,                      \
 			       BT_GATT_CHRC_READ | BT_GATT_CHRC_NOTIFY,        \
-			       BT_GATT_PERM_READ_ENCRYPT,                      \
+			       BT_GATT_PERM_READ_ENCRYPT | BT_GATT_PERM_READ_128B_ENTROPY,   \
```

Comment:
Do we need need to have both `READ_ENCRYPT` and `READ_128B_ENTROPY`? Shouldn't the latter include encryption given the documentation: `Attribute write permission with 128bit entropy key connection.`?

Reply:
We don't, you are right about this, `READ_128B_ENTROPY` is for sure a subset of `READ_ENCRYPT`. We should also adjust the permission checks accordingly.

Reply:
`128B_ENTROPY` is indeed a superset of `ENCRYPT`, similar to how `AUTHEN` is a superset of `ENCRYPT`, but `128B_ENTROPY` is _not_ a superset of `AUTHEN`, so perhaps the GATT documentation should also be updated to reflect that?

Reply:
Would you think this would suffice in the description?

	/** @brief Attribute write permission with 128bit entropy key connection.
	 *
	 *  If set, write access requires connection using an at least 128bit entropy
	 *  key derived from one of the following sources:
	 *  - LE Secure Connections
	 *  - OOB method
	 * 
	 *  This permission is a superset of encryption, meaning if set
	 *  BT_GATT_PERM_WRITE_ENCRYPT is implied, and can be omitted. It does
	 *  however not imply BT_GATT_PERM_WRITE_AUTHEN.
	 */

Reply:
Yeah, something like that. 

Similar text should/could be added to `BT_GATT_PERM_WRITE_AUTHEN`, as that is also missing, right?